config/environments/production.rb

```diff
@@ -60,14 +47,18 @@
     }
   }
 
-  # Prepend all log lines with the following tags.
+  # Log to STDOUT with the current request id as a default log tag.
   config.log_tags = { request_id: :request_id }
 
-  # "Info" include generic and useful information about system operation, but avoids logging too much
-  # information to avoid inadvertent exposure of personally identifiable information (PII). If you
-  # want to log everything, set the level to "debug".
+  # Change to "debug" to log everything (including potentially personally-identifiable information!)
   config.log_level = ENV.fetch("RAILS_LOG_LEVEL", "info")
 
+  # Prevent health checks from clogging up the logs.
+  config.silence_healthcheck_path = "/up"
```

Comment:
Fair enough, I was trying to keep these files as close to standard Rails as possible, I might put it in `environments/development.rb` instead of `application.rb`.